src/com/facebook/buck/go/GoDescriptors.java

```diff
@@ -33,9 +41,24 @@
 import com.google.common.collect.ImmutableSortedSet;
 
 import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.List;
 
 abstract class GoDescriptors {
+  private static final Path PATH_TO_TEST_GEN =
```

Comment:
This is how we used to add resources in the past, now we specify them in `resources` in the `java_library` in the BUCK file.

See `build-with-buck.st` for example.